tox.ini

```diff
@@ -15,7 +15,6 @@ deps =
   -r{toxinidir}/requirements-dev.txt
 whitelist_externals = make
                       bash
-                      coverage
                       nosetests
```

Comment:
I think the whole `whitelist_externals` section is unneeded. It's amazing how many places this code has propagated 😩